transport/http/handler.go

```diff
@@ -147,13 +149,11 @@ func handleOnewayRequest(
 	return nil
 }
 
-func updateSpanWithErr(span opentracing.Span, err error) error {
+func updateSpanWithErr(span opentracing.Span, err error) {
```

Comment:
One of the calls was done without using the return value, so you would have to do `_ = updateSpanWithErr(span, err)`